.travis.yml

```diff
@@ -40,5 +40,5 @@ addons:
 before_install:
 
 script:
-  - go test -v -race ./...                   # Run all the tests with the race detector enabled
+  - go test -v -race -mod=readonly ./...     # Run all the tests with the race detector enabled
```

Comment:
It looks like this doesn't work for the older releases that travis ci is running.

Reply:
I'd drop go1.10 because it doesn't support modules and add go1.13, what do you think on that?

Reply:
Sounds fine to me.

Reply:
Pushed the change, seems like CI jobs are passing now

Reply:
Thanks!